src/app/[locale]/settings/_components/mnestix-connections/MnestixConnectionsCard.tsx

```diff
@@ -6,7 +6,7 @@ import { useNotificationSpawner } from 'lib/hooks/UseNotificationSpawner';
 import {
     getConnectionDataAction,
     upsertConnectionDataAction,
-} from 'lib/services/repository-access/MnestixConnectionServerActions';
+} from 'lib/services/connection-access/mnestixConnectionServerActions';
```

Comment:
In our context, connection mostly refers to the connection settings, where we can configure multiple repositories etc. In this case it's the connection to the database and I would just call it database-access